docs/1-introduction/1-1-getting-started/1-1-2-hello-d3d11.md

```diff
@@ -205,15 +205,18 @@ typedef struct _GUID
 ```
 GUID's essentially are an arbitrary but unqiue value that repisent a Microsoft Interface Definition Language (MIDL) type, which breaks down into 2 things:
 - a single COM object/interface — the backbone to every ID3D11XXX type  
-- a single entry-point vector (EPV) — a unique array of function pointers geared towards Remote Procedure Call's (RPC) which is out of scope for d3d11 but can be learned more if read about in MSDN documentation (https://docs.microsoft.com/en-us/windows/win32/rpc/registering-interfaces#entry-point-vectors)
+- a single Entry-Point Vector (EPV) — a unique array of function pointers geared towards Remote Procedure Call's (RPC) which is out of scope for d3d11 but can be learned more if read about in MSDN documentation (https://docs.microsoft.com/en-us/windows/win32/rpc/registering-interfaces#entry-point-vectors)
 
-What the parts of the `IID_PPV_ARGS(ppType)` macro do: 
+What the parts of the `IID_PPV_ARGS(ppType)` macro are: 
+
+[the `ppType` in `IID_PPV_ARGS(ppType)`]
+- a pointer to a pointer of a object.
 
 [the `__uuidof(**(ppType))` part of `IID_PPV_ARGS(ppType)`]
--  at compile time retrives a `UUID` which repisents a `GUID`, which is returned as a a `REFIID` — which means that the type returned is a reference to an identifier to a type of COM object (in the scope of D3D11).   
+-  at compile time retrives a `UUID` from `ppType` type which represents a `GUID`, which is returned as a a `REFIID` — which means that the type returned is a reference to an identifier to a specific type of COM object.   
 
 [the `IID_PPV_ARGS_Helper(ppType)` part of `IID_PPV_ARGS(ppType)`]
-- `IID_PPV_ARGS_Helper` casts your inputted pointer to a COM type to a `void**` and returns it so that your inputted variable can be set to the `[out]` parameter of the function you used `IID_PPV_ARGS(ppType)` in to retrive the output of the function, like the created `IDXGIFactory2` in `CreateDXGIFactory2`.
+- `IID_PPV_ARGS_Helper` casts the inputted ``ppType` to a `void**` and returns it so that `ppType` can be set to the `[out]` parameter of the function `IID_PPV_ARGS(ppType)` was used in to. This being done to retrive the output of the function, like with the created `IDXGIFactory2` in `CreateDXGIFactory2`.
```

Comment:
could I instead just put the function definition to show what IID_PPV_ARGS_Helper(ppType) does specifically — since I think its good to know that IID_PPV_ARGS is not a black box made from a black box

Reply:
we dont use the Helper variant anywhere at all, if people want to dig deeper into the IID_PPV_ARGS rabbit hole they can just F12 themselves or read the docs. We don't have to inflate this helper macro any larger than it actually is. K.I.S.S.

Reply:
should I remove the `What the parts of the IID_PPV_ARGS(ppType) macro are:` section?

Reply:
Nah that should be ok, as part of IID_PPV_ARGS. Just get rid of the Helper and the typos :P then it  should be good to go.

Reply:
with the latest changes I think I explain everything you asked for - and removed or explained what was asked by others 

> It does not "compute", should be: "
the GUID explanation was in depth for this request - although it was in the end removed